### PR TITLE
Added server side AWS account ID log redaction

### PR DESCRIPTION
**Pull request by micahhausler**
This feature allows administrators to prevent specific account IDs from being emitted in server logs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -394,6 +394,11 @@ server:
   # role to assume before querying EC2 API in order to discover metadata like EC2 private DNS Name
   ec2DescribeInstancesRoleARN: arn:aws:iam::000000000000:role/DescribeInstancesRole
 
+  # AWS Account IDs to scrub from server logs. (Defaults to empty list)
+  scrubbedAccounts:
```
Comment (nckturner) [inline README.md:398]: Nit: To me "redacted" is more explicit than "scrubbed"

File: README.md [hunk 2]
```diff
@@ -394,6 +394,11 @@ server:
   # role to assume before querying EC2 API in order to discover metadata like EC2 private DNS Name
   ec2DescribeInstancesRoleARN: arn:aws:iam::000000000000:role/DescribeInstancesRole
 
+  # AWS Account IDs to scrub from server logs. (Defaults to empty list)
```
Comment (nckturner) [inline README.md:397]: Do we want to give the ability to redact more than just account IDs?

File: README.md (hunk 1 above)
Comment (micahhausler) [reply README.md:398]: I went back and forth on this, but in my mind "redacted" sounded too much like the whole log entry was omitted, and in this case I'm just redacting part of the log line.

File: README.md (hunk 2 above)
Comment (micahhausler) [reply README.md:397]: I think we could certainly add that at some point

File: README.md (hunk 1 above)
Comment (wongma7) [reply README.md:398]: I like and understand "scrub" FWIW. 👍

**Pull request merged**